tests/system/src/test/java/test/javafx/scene/input/ClipboardTest.java

```diff
@@ -33,14 +34,18 @@
 import org.junit.Test;
 import test.util.Util;
 
-import java.awt.*;
+import java.awt.Toolkit;
 import java.awt.datatransfer.DataFlavor;
 import java.awt.datatransfer.StringSelection;
+import java.awt.datatransfer.Transferable;
 import java.util.concurrent.CountDownLatch;
 import java.util.concurrent.TimeUnit;
 
-import static junit.framework.TestCase.fail;
-import static org.junit.Assert.assertEquals;
+import sun.awt.datatransfer.ClipboardTransferable;
+import sun.awt.datatransfer.SunClipboard;
+
+import static org.junit.Assert.*;
```

Comment:
I am not sure about how strict we are about using wildcard imports in tests in JavaFX. You can change this or keep it as it is, depending upon the answer to the first statement.

Reply:
In general wildcard imports are discouraged; static imports (e.g., `Assert.*`) are an exception used in many of our tests.